lib/util.h

```diff
@@ -220,6 +220,9 @@ extern char *create_tempdir(const char *path, const char *subname);
  * value of remove on error. */
 extern int removedir(const char *path);
 
+/* Call rename but fsync the directory before returning success */
+extern int cyrus_rename(const char *src, const char *dest);
```

Comment:
`extern ` is not necessary.

Reply:
sure but it's present on ALL THE OTHER THINGS NEARBY; so I'm gonna be consistent with the surrounding code, because it will be less weird for anyone cleaning it up later.